ButterflyMX Demo/Controllers/LoginViewController.swift

```diff
@@ -76,6 +68,23 @@ class LoginViewController: UITableViewController {
                    clientID: Bundle.main.object(forInfoDictionaryKey: "butterflymx-CLIENTID") as? String ?? "N/a")
         SVProgressHUD.setDefaultStyle(.light)
     }
+    
+    private func getBMXAuthProvider(for environment: BMXBackendEnvironment) -> BMXAuthProvider {
+        switch environment {
+        case .development:
```

Comment:
you repeat there the same code 3 times just with different keys, this also can be excluded to a function